data/CosmosDbSeeder/items/dev/core_data/site_ABC03.json

```diff
@@ -0,0 +1,52 @@
+{
+  "id": "e1a9f96e-b86c-49ef-9ca5-613d596c09ad",
+  "docType": "site",
+  "name": "Lloyds Pharmacy",
+  "address": "Batley, Leeds, LS34 7AG",
+  "phoneNumber": "0113 333333",
+  "odsCode": "ABC02",
+  "region": "R3",
+  "integratedCareBoard": "ICB3",
+  "location": {
+    "type": "Point",
+    "coordinates": [-1.67382134, 55.79628754]
+  },
+  "attributeValues": [
```

Comment:
I noticed on my playwright refactor PR that this is wrong. The other two sites were updated but this wasn't